package-overrides/npm/@angular/core@2.0.0-rc.1.json

```diff
@@ -1,7 +0,0 @@
-{
-  "meta": {
-    "src/*.js": {
-      "deps": ["reflect-metadata", "zone.js"]
-    }
-  }
-}
```

Comment:
This file can be left here, to support existing users. The new one without the prefix will still take preference.